cypress/e2e/4-node-creator.cy.ts

```diff
@@ -24,16 +22,7 @@ describe('Node Creator', () => {
 				res.send(nodes)
 			})
 		}).as('nodesIntercept')
-
-		cy.signup(username, firstName, lastName, password);
-
-		cy.on('uncaught:exception', (err, runnable) => {
-			expect(err.message).to.include('Not logged in');
-
-			return false;
-		})
-
-		cy.signin(username, password);
+		cy.task('db:reset');
```

Comment:
As a precaution, resetting the DB should be the first instruction in `beforeEach`.

Reply:
In the specs, there's no saving of the workflow, or user management etc. So its seems safe to just do it once before running all the tests; fewer operations to do > faster spec.